pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1392,131 +1609,1374 @@ def marker_minimum(self):
+        default ratio is 1. When a new ratio is selected, the video bandwidth changes to satisfy the
+        new ratio—the resolution bandwidth does not change value.
+        """,
+        validator=strict_range,
+        values=arange(0.002, 0.10, 0.001)
+    )
+
+    def view_trace(self, trace):
+        """Display the current contents of the selected trace, but does not update
+        the contents. View mode may be executed before a sweep is complete when :meth:`single_sweep`
+        and :meth:`trigger_sweep` are not used.
+
+        :param trace: A representation of the trace, either from :class:`Trace` or
+            use 'TRA' for Trace A or 'TRB' for Trace B
+        :type trace: str
+        :raises TypeError: Type isn't 'string'
+        :raises ValueError: Value is 'TRA' nor 'TRB'
+        """
+        if not isinstance(trace, str):
+            raise TypeError("Should be of type string but is '%s'" % type(trace))
+
+        if trace not in [e for e in Trace]:
+            raise ValueError("Only accepts values of [%s] but was '%s'" % ([e for e in Trace],
+                                                                           trace))
+        self.write("VIEW " + trace)
+
+    video_trigger_level = Instrument.control(
+        "VTL?", "VTL %.3f",
+        """
+        Control the video trigger level when the trigger mode is set to VIDEO (refer
+        to the :attr:`trigger_mode` command). A dashed line appears on the display to indicate the
+        level. The default value is 0 dBm. Range -220 to 30.
+
+        Type: :code:`float`
+        """,
+        validator=strict_range,
+        values=[-220, 30]
+    )
+
+
+class HP8560A(HP856Xx):
+    """Represents the HP 8560A Spectrum Analyzer and provides a high-level
+    interface for interacting with the instrument.
+
+    .. code-block:: python
+
+        from pymeasure.instruments.hp import HP8560A
+        from pymeasure.instruments.hp.hp856Xx import AmplitudeUnits
+
+        sa = HP8560A("GPIB::1")
+
+        sa.amplitude_unit = AmplitudeUnits.DBUV
+        sa.start_frequency = 299.5e6
+        sa.stop_frequency = 300.5e6
+
+        print(sa.marker_amplitude)
+    """
+
+    # HP8560A is able to go up to 2.9 GHz
+    MAX_FREQUENCY = 2.9e9
+
+    def __init__(self, adapter, name="Hewlett-Packard HP8560A", **kwargs):
+        super().__init__(
+            adapter,
+            name,
+            **kwargs,
+        )
+
+        self.center_frequency_values = [0, self.MAX_FREQUENCY]
+        self.start_frequency_values = [0, self.MAX_FREQUENCY]
+        self.stop_frequency_values = [0, self.MAX_FREQUENCY]
+        self.frequency_offset_values = [0, self.MAX_FREQUENCY]
+        self.marker_frequency_values = [0, self.MAX_FREQUENCY]
+        self.span_values = [["FULL", "ZERO"], [0, self.MAX_FREQUENCY]]
+
+    source_leveling_control = Instrument.control(
+        "SRCALC?", "SRCALC %s",
+        """
+        Select internal or external leveling for use with the
```

Comment:
Please start the docstrings of all `control`s with "Control". Of all `measurement` with "Measure" or "Get" and all `setting` with "Set".

Reply:
Today I merged a PR, which checks all properties to have correct docstrings.

Please merge Master into your branch to activate the test.